systest/mutations_test.go

```diff
@@ -1733,7 +1733,7 @@ func CountIndexConcurrentSetDelUIDList(t *testing.T, c *dgo.Dgraph) {
 				if atomic.AddUint64(&txnCur, 1) > txnTotal {
 					break
 				}
-				id := 2 + int(r.Int31n(int32(maxUID))) // 1 id subject id.
+				id := 2 + int(rand.Int31n(int32(maxUID))) // 1 id subject id.
```

Comment:
You're using the global rand which has locks and it will cause contention.

Reply:
yeah I understand but rand.NewSource is not concurrent safe. And we were having race condition here. This is test case so I thought it would be fine.